src/Command/SendTeachingRemindersCommand.php

```diff
@@ -151,32 +165,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $daysInAdvance = $input->getOption('days');
+        $daysInAdvance = (int) $input->getOption('days');
         $sender = $input->getArgument('sender');
         $baseUrl = rtrim($input->getArgument('base_url'), '/');
         $subject = $input->getOption('subject');
         $isDryRun = $input->getOption('dry-run');
         $senderName = $input->getOption('sender_name');
+        $schools = $input->getOption('schools');
         $from = $sender;
         if ($senderName) {
             $from = [$sender => $senderName];
         }
+        if ($schools) {
+            $schoolIds = array_map('intval', str_getcsv($schools));
```

Comment:
`str_getcsv()` is the new `explode()`? spicy... :fire: :fire:

Reply:
Yea, I went with it because then I didn't have to filter the whitespace out of `1,   2,     3`.

Reply:
makes sense. i'm adding it to my arsenal of obscure but useful functions.